### PR TITLE
Update REVIEWING doc with changes to LGTM process

### PR DESCRIPTION
**Pull request by pmorie**
Per the July 10 2017 meeting.

### CODE REVIEW

File: REVIEWING.md [hunk 1]
```diff
@@ -7,9 +7,7 @@ situations for which process is not specified.
 
 PRs may only be merged after the following criteria are met:
 
-1. It has been open for at least 1 business day
-1. It has no `veto` label on it
-1. It has been 'LGTM'-ed by 3 different reviewers, each from a different
+1. It has been 'LGTM'-ed by 2 different reviewers, each from a different
   organization
```
Comment (duglin) [inline REVIEWING.md]: and different from the author's org.

**Comment by MHBauer** [comment]
@pmorie I was out, where are the videos posted? I looked in the kube channel on youtube and the most recent is from January, but I feel like I've seen more recent ones.

**Comment by MHBauer** [comment]
Found it, personal account, https://www.youtube.com/watch?v=JRehJdb-xxc

**Comment by pmorie** [comment]
@MHBauer there's actually a playlist for kubernetes that they're part of, which is linked to from the readme in this repo.

**Comment by pmorie** [comment]
@duglin nit addressed

**Comment by duglin** [comment]
thanks LGTM

**Pull request merged**